src/components/comment/comment.test.ts

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import type { PleasantestUtils } from 'pleasantest';
+import { withBrowser } from 'pleasantest';
+import { loadTwigTemplate } from '../../../test-utils';
+
+const commentMarkup = loadTwigTemplate(path.join(__dirname, 'comment.twig'));
+const initCommentsJs = (utils: PleasantestUtils) =>
+  utils.runJS(`
+    import { initComments } from './comment';
+    initComments();
+  `);
+
+test(
+  'reply form can be opened and closed',
+  withBrowser(async ({ utils, screen, user }) => {
+    await utils.injectHTML(
+      await commentMarkup({
+        comment: {
+          ID: 1,
+          link: '#comment-1',
+          date: new Date(),
+          avatar: '',
+          author: {
+            name: 'Test',
+          },
+          comment_content: 'Test',
+          approved: true,
+        },
+        allow_replies: true,
+      })
+    );
+
+    await utils.loadCSS('../../../dist/standalone.css');
```

Comment:
Would it make sense to extract this into a function in the test-utils file? The same thing is happening in the sky nav tests and the elastic textarea tests

Reply:
Yeah, that's a good idea! I'll do that/